content/blog/quick-kafka-installation/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Quick Kafka Installation on Windows"
+date: "2018-08-24"
+coverImage: "messagelog.png"
+author: "Ashish Sharma"
+tags: ["Kafka"]
```

Comment:
you can add tags as ["Kafka", "Windows"]

Reply:
Done